pkg/csi/service/wcp/controller.go

```diff
@@ -611,9 +615,24 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		ContentSourceSnapshotID: contentSourceSnapshotID,
 	}
 
-	volumeInfo, faultType, err := common.CreateBlockVolumeUtil(ctx, cnstypes.CnsClusterFlavorWorkload,
-		c.manager, &createVolumeSpec, candidateDatastores, filterSuspendedDatastores,
-		isTKGSHAEnabled, checkCompatibleDataStores)
+	var (
+		volumeInfo *cnsvolume.CnsVolumeInfo
+		faultType  string
+	)
+	if isPodVMOnStretchSupervisorFSSEnabled {
+		volumeInfo, faultType, err = common.CreateBlockVolumeUtil(ctx, cnstypes.CnsClusterFlavorWorkload,
+			c.manager, &createVolumeSpec, candidateDatastores, filterSuspendedDatastores,
+			isTKGSHAEnabled, checkCompatibleDataStores,
+			cnsvolume.CreateVolumeExtraParams{
```

Comment:
don't we need to supply pointer for CreateVolumeExtraParams here?

Reply:
Changed it